content/pt-br/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Atribuindo Pods aos Nós Usando Afinidade de Nós
+min-kubernetes-server-version: v1.10
+content_type: task
+weight: 120
+---
+
+<!-- overview -->
+Esta página mostra como atribuir um Pod Kubernetes à um Nó particular usando 
+Afinidade de Nó em um cluster Kubernetes.
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+<!-- steps -->
+
+## Adicione um rótulo a um Nó
+
+1. Liste os Nós em seu cluster, juntamente com seus rótulos:
+
+    ```shell
+    kubectl get nodes --show-labels
+    ```
+    A saída é similar a esta:
+
+    ```shell
+    NAME      STATUS    ROLES    AGE     VERSION        LABELS
+    worker0   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker0
+    worker1   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker1
+    worker2   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker2
+    ```
+
+1. Escolha um dos seus Nós, e adicione um rótulo a ele:
+
+    ```shell
+    kubectl label nodes <your-node-name> disktype=ssd
+    ```
+    onde `<your-node-name>` é o nome do seu nó escolhido.
+
+1. Verifique se o seu nó escolhido tem o rótulo `disktype=ssd`:
+
+    ```shell
+    kubectl get nodes --show-labels
+    ```
+
+    A saída é similar a esta:
+
+    ```
+    NAME      STATUS    ROLES    AGE     VERSION        LABELS
+    worker0   Ready     <none>   1d      v1.13.0        ...,disktype=ssd,kubernetes.io/hostname=worker0
+    worker1   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker1
+    worker2   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker2
+    ```
+
+    Na saída anterior, você pode ver que o nó `worker0` tem o rótulo `disktype=ssd`.
+
+## Agende um Pod usando afinidade de Nó requerida
+
+Este manifesto descreve um Pod que tem uma afinidade de nó 
+`requiredDuringSchedulingIgnoredDuringExecution`,`disktype: ssd`. 
+Isto significa que o pod será agendado somente em um nó que tenha o rótulo `disktype=ssd`. 
```

Comment:
```suggestion
## Agende um Pod usando afinidade de nó requerida

Este manifesto descreve um Pod que tem uma afinidade de nó 
do tipo `requiredDuringSchedulingIgnoredDuringExecution`,
com o seletor de rótulo `disktype: ssd`. 
Isto significa que o pod será agendado somente em um nó que tenha o rótulo `disktype=ssd`. 
```

I know this is how it is described in the English version, but it sounded a bit strange to me in Portuguese.